### DISCUSSION TITLE
Clickhouse stats after query consulting

### DISCUSSION THREAD

**Discussion by calebeaires** [depyronick/clickhouse-client#19] [Q&A]
Some javascript clickhouse delivery some stats on each response. Those can be very useful. After the response is processed, whe can read a supplemental response data from it, such as row count and others. Here is an example:

![CleanShot 2022-08-14 at 16 34 15](https://user-images.githubusercontent.com/794195/184552104-4e9cb79b-41ff-4cb9-a37d-55bf2029c493.png)

Is it possible to get the same result with clickhouse-client library?

**Comment by depyronick** [comment]
hi

It’s planned.
